src/main/java/com/denizenscript/depenizen/bukkit/properties/mythicmobs/MythicMobsEntityProperties.java

```diff
@@ -74,6 +77,21 @@ public static void registerTags() {
             }
             return null;
         }, "mythic_mob");
+
+        // <--[tag]
+        // @attribute <EntityTag.parse_mythic[<placeholder_text>]>
+        // @returns ElementTag
+        // @plugin Depenizen MythicMobs
+        // @description
+        // Parses Mythic placeholders and returns the result. The target scope is set as the entity. (For example, <target.name> returns the entity's name.) Keep in mind that <> will have to be escaped using tags. This is usually over-engineering, consider using other Denizen tags instead.
+        // -->
+        PropertyParser.registerTag(MythicMobsEntityProperties.class, ElementTag.class, "parse_mythic", (attribute, object) -> {
```

Comment:
should use the required-param syntax of registerTag (add an additional `, ElementTag.class` and make the interface like `(attribute, object, text) ->`